server/ai_process.go

```diff
@@ -1043,10 +1044,30 @@ func submitLiveVideoToVideo(ctx context.Context, params aiRequestParams, sess *A
 		if err != nil {
 			return nil, fmt.Errorf("control pub url - %w", err)
 		}
+		controlStr := control.String()
+
 		clog.V(common.VERBOSE).Infof(ctx, "pub %s sub %s control %s", pub, sub, control)
 		startTricklePublish(pub, params)
 		startTrickleSubscribe(sub, params)
 		startControlPublish(control, params)
+
+		//Switch subscribe and publish urls
+		req.SubscribeUrl = pub.String() // publish url
+		req.PublishUrl = sub.String() // subscriber url
```

Comment:
Are these purposefully inverted? Sub = pub and Pub = sub. Maybe it's necessary since the runner args have a different "perspective"

Reply:
Correct, the initial values have `-out` as the subscriber url on the orchestrator so it is necessary to swap these so the ai-runner publishes to `-out`